src/GitHook/Command/FileCommand/PreCommit/PhpStanCheckCommand.php

```diff
@@ -77,14 +72,43 @@ public function run(CommandContextInterface $context)
      */
     protected function isFileAllowed(string $filePath, array $allowedDirectories): bool
     {
-        $fileDirectory = dirname(realpath($filePath));
+        $fileDirectory = dirname($filePath);
 
         foreach ($allowedDirectories as $allowedDirectory) {
-            if (strpos($fileDirectory, realpath($allowedDirectory)) === 0) {
+            if (strpos($fileDirectory, $allowedDirectory) !== false) {
                 return true;
             }
         }
 
         return false;
     }
+
+    /**
+     * @param \GitHook\Command\FileCommand\PreCommit\PhpStan\PhpStanConfiguration $phpStanConfiguration
+     * @param \GitHook\Command\Context\CommandContextInterface $context
+     *
+     * @return int
+     */
+    protected function getLevel(PhpStanConfiguration $phpStanConfiguration, CommandContextInterface $context): int
+    {
+        $level = $phpStanConfiguration->getLevel();
+
+        if (!$context->isModuleFile()) {
```

Comment:
This should be removed and instead getModulePath should be named getPath() and do the auto resolvement.
Rest all good.

Reply:
Only as a follow-up, we have to talk about it. If nothing else from your side, merge, please.

Reply:
This sounds like "wont happen", which would be bad. The actual interface now already contains a method we will have to throw out again with the next commit. Not sure if separate fixes make sense here, when it is so simple to resolve, right?